### PR TITLE
Email teams regarding competition dates & robot celebrations

### PR DESCRIPTION
**Pull request by PeterJCLaw**
Fixes https://github.com/srobo/competition-team-minutes/issues/221

See also https://github.com/srobo/website/pull/211

### CODE REVIEW

File: SR2020/2020-06-25-virtual-competition-dates-and-robot-celebrations.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+prior to the transition to a virtual competition. We appreciate that many teams
+will not have had access to their robots during the lock-down, so will not be
+awarding prizes for this, however we would still like to recognise the effort
+which the teams had put in.
+
+We would like to hear from you if your team has any designs, photos or video of
+your physical robots which you would like to share for inclusion in the live
+stream broadcast.
+
+We are also interested in recording remote interviews with team members
```
Comment (RealOrangeOne) [inline SR2020/2020-06-25-virtual-competition-dates-and-robot-celebrations.md:38]: Is there a CTA for this bit?

Comment (PeterJCLaw) [reply SR2020/2020-06-25-virtual-competition-dates-and-robot-celebrations.md:38]: Apparently not. Should we just ask them to email teams@?

Comment (RealOrangeOne) [reply SR2020/2020-06-25-virtual-competition-dates-and-robot-celebrations.md:38]: Yeah emailing works.

Also, The requirement of media consent will mean updating our template.

Comment (PeterJCLaw) [reply SR2020/2020-06-25-virtual-competition-dates-and-robot-celebrations.md:38]: Hadn't thought about template update. Pretty sure we'd need the consent though?

Comment (PeterJCLaw) [reply SR2020/2020-06-25-virtual-competition-dates-and-robot-celebrations.md:38]: anyway: 4e559cc2c9c05465332536aae70723952013e148

**Comment by RealOrangeOne** [comment]
Sent

**Pull request merged**